lsp-completion.el

```diff
@@ -710,8 +708,8 @@ See #2675"
   (cl-assert item nil "Completion item must not be nil")
   (lsp-completion--fix-resolve-data item)
   (or (ignore-errors
-        (when (lsp-feature? "completionItem/resolve")
-          (lsp-request "completionItem/resolve" item)))
+        (when (lsp-feature? "textDocument/completionResolve")
```

Comment:
I don't see such method in the spec.

Reply:
Thanks for noticing, my refactoring has gone too far :D